go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/ceph/ceph-csi-operator
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.5
+toolchain go1.23.8
```

Comment:
can we remove toolchain is it required? i dont see that we have it in go.mod in main branch

Reply:
Yes, we can remove it. It just denotes the preferred golang version.